src/main/java/org/openrefine/extensions/commons/importer/FileRecordToRows.java

```diff
@@ -0,0 +1,30 @@
+package org.openrefine.extensions.commons.importer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.refine.importers.TabularImportingParserBase.TableDataReader;
+
+public class FileRecordToRows implements TableDataReader {
+    List<Object> rowsOfCells;
+
+    Iterator<FileRecord> iteratorFileRecords;
+
+    public FileRecordToRows(Iterator<FileRecord> iteratorFileRecords) {
+        this.iteratorFileRecords = iteratorFileRecords;
+    }
+    @Override
+    public List<Object> getNextRowOfCells() throws IOException {
+        rowsOfCells = new ArrayList<>();
+        rowsOfCells.add(iteratorFileRecords.next());
```

Comment:
The `List<Object>` that this method should return is meant to be the next row, represented as a list of values (for each column in the project). Here, you are returning a single value: the `FileRecord` object that came from the fetching code. But a `FileRecord` object does not correspond to a single cell: it contains the file name, the mid, and the categories, each of which should go into a different cell.